packages/form-upload/src/FileRow.js

```diff
@@ -61,7 +61,7 @@ const FileRow = ({ onRemove, children, file }) => {
         </div>
       </td>
       <td className="align-middle" style={{ width: '45%' }}>
-        <UploadProgressBar upload={file} />
+        <UploadProgressBar upload={file} aria-label={`${file.file.name} upload`} />
```

Comment:
should we add `progress` to the end of the label or is that implied somehow?

Reply:
the screenreader already reads the role of progressbar aloud and if traversing left/right between the cells also reads the column header of "upload progress"